projects/hslayers/src/common/endpoints/endpoint.interface.ts

```diff
@@ -20,7 +20,7 @@ export interface HsEndpoint {
   language?;
   listLoading?;
   layers?: HsAddDataLayerDescriptor[];
-  originalConfiguredUser?;
+  originalConfiguredUser?: 'anonymous';
```

Comment:
This can make us some pain later. Try restricting the type of `datasources` in the config object to `HsEndpoint[]` (which it really should be), i.e. here https://github.com/hslayers/hslayers-ng/blob/2aa278003150e84e0c922cbb9ff3fca9594a95c8/projects/hslayers/src/config.service.ts#L129

The tsc compiler will scream about incompatible types, where in the lengthy message the last lines are important:

> Types of property 'originalConfiguredUser' are incompatible.
        Type 'string' is not assignable to type '"anonymous"'

i.e. it would require the syntax `originalConfiguredUser: 'anonymous' as const` everywhere.

In that context I wonder if we do need the `originalConfiguredUser` param at all... 💭

Reply:
>In that context I wonder if we do need the originalConfiguredUser param at all... 💭
Nope